frontend/app/components/redesign/components/ToolsDropdown.tsx

```diff
@@ -85,7 +87,7 @@ export const ToolsDropdown = forwardRef<HTMLDivElement, ToolsDropdownProps>(
     }
 
     return (
-      <div ref={ref} className={cx('space-y-3xs', className)}>
+      <div key={uniqueId} ref={ref} className={cx('space-y-3xs', className)}>
```

Comment:
This is unnecessary. The `key` coming from `useId` will remain constant across renders.